.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
     - wget http://bit.ly/miniconda -O miniconda.sh
     - bash miniconda.sh -b -p $HOME/miniconda
     - export PATH="$HOME/miniconda/bin:$PATH"
+    - conda config --add channels conda-forge -f
```

Comment:
This ensures the latest `cell_tree2d` which fixes the segfaults I was experimenting.